docs/installation/viam-micro-server-dev.md

```diff
@@ -55,240 +55,229 @@ If you haven't already, install Homebrew:
-   - Select the part status dropdown to the right of your machine's name on the top of the page: {{<imgproc src="configure/machine-part-info.png" resize="500x" declaredimensions=true alt="Restart button on the machine part info dropdown">}}
-   - Click the copy icon underneath **Machine cloud credentials**.
-     `viam-micro-server` needs this JSON, which contains your machine part secret key and cloud app address, to connect to the [Viam app](https://app.viam.com).
-   - Paste the machine cloud credentials into your terminal when prompted.
+   Compile the project for ESP32:
 
-   {{% snippet "secret-share.md" %}}
+   ```sh { class="command-line" data-prompt="$"}
+   make build-esp32-bin
+   ```
 
-1. Upload `viam-micro-server` to your ESP32
+   Please note that the first build may be fairly time consuming, as ESP-IDF must be cloned and built, and all dependent Rust crates must be fetched and built as well.
+   Subsequent builds will be faster.
 
-   Now, flash the project to your ESP32 and it will connect to [the Viam app](https://app.viam.com) from which you can then remotely control it:
+   Next, flash the generated firmware to your ESP32.
 
-   {{< tabs >}}
-   {{% tab name="Use Canon" %}}
+   Connect the ESP32 board you wish to flash to a USB port on your development machine, and run:
 
-```sh { class="command-line" data-prompt="$"}
-cd <your-path-to/your-project-directory>
-canon bash -lc "make build-esp32-bin"
-make flash-esp32-bin
-```
+   ```sh { class="command-line" data-prompt="$"}
+   make flash-esp32-bin
+   ```
 
-    {{% /tab %}}
-    {{% tab name="Local environment" %}}
+   When prompted, select the serial port that your ESP32 is connected to through a data cable.
 
-Make sure you have run `. ~/dev/esp/export-rs.sh` and `. ~/dev/esp/esp-idf/export.sh` before running the following command:
+   {{% /tab %}}
+   {{% tab name="Run Natively" %}}
 
-```sh { class="command-line" data-prompt="$"}
-make upload
-```
+   The native server can be built and run with one command:
 
-    {{% /tab %}}
-    {{% /tabs %}}
+   ```sh { class="command-line" data-prompt="$"}
+   make native
+   ```
 
-    When prompted, select the serial port that your ESP32 is connected to through a data cable.
+   {{% /tab %}}
+   {{% /tabs %}}
 
-    If successful, you will retain a serial connection to the board until you press `Ctrl-C`.
-    To manage this connection, consider running it within a dedicated terminal session, or under `tmux` or `screen`.
-    While the serial connection is live, you can also restart the currently flashed image with `Ctrl-R`.
+{{% /tab %}}
+{{% /tabs %}}
 
-1. Navigate to your new machine's page on [the Viam app](https://app.viam.com).
-   If successful, **Live** should be displayed underneath **Last online**.
+For further details on `viam-micro-server` development, including credentials management and developer productivity suggestions, please see the (development technical notes page on GitHub)[TBD].
```

Comment:
@acmorrow Is there a link I can add here or do we want to leave this out for now?

Reply:
The link can be populated when https://github.com/viamrobotics/micro-rdk/pull/320 is merged.

Reply:
Ok thank you, I'll keep an eye on that! It looks like it's about to merge so I'll keep on eye on that the next few days and try to merge this with the link

Reply:
It merged, the link is https://github.com/viamrobotics/micro-rdk/blob/main/DEVELOPMENT.md

Reply:
Thank you! Adding in